vendor/bat-native-ledger/src/bat/ledger/internal/wallet/wallet.cc

```diff
@@ -162,27 +161,17 @@ void Wallet::DisconnectWallet(
 }
 
 void Wallet::ClaimFunds(ledger::ResultCallback callback) {
-  // Anonymous funds claim
-  claim_->Start([this, callback](const type::Result result) {
-    if (result != type::Result::LEDGER_OK &&
-        result != type::Result::ALREADY_EXISTS) {
-      BLOG(0, "Claiming anon funds failed");
-      callback(type::Result::CONTINUE);
-      return;
-    }
-
-    // tokens claim
-    ledger_->promotion()->TransferTokens(
-        [callback](const type::Result result, std::string drain_id) {
-          if (result != type::Result::LEDGER_OK) {
-            BLOG(0, "Claiming tokens failed");
-            callback(type::Result::CONTINUE);
-            return;
-          }
+  // tokens claim
+  ledger_->promotion()->TransferTokens(
```

Comment:
Since we are no longer performing "user funds" claiming in this method, we should probably either:

- Rename the method (e.g. `TransferTokens`), or
- Remove this method and perform the call to `ledger_->promotion()->TransferTokens` directly in `uphold_wallet.cc` (similar to how it's done in `bitflyer_wallet.cc`).

Reply:
Resolved in 088ad1c7ccdaa1ed422fbc903352a0a5f236ef6b.